edflow/eval/pipeline.py

```diff
@@ -292,6 +296,10 @@ def after_epoch(self, epoch):
 
             kwargs = cb_kwargs.get(n, {})
             results[n] = cb(self.root, self.data_in, data_out, self.config, **kwargs)
+        if self.clean_after_callbacks:
+            self.logger.info(
+                "Cleaning up evaluation data at {}".format(self.save_root))
+            shutil.rmtree(self.save_root, ignore_errors=True)
```

Comment:
How save is this? Could one set save_root to be the project root? Should we check this? 
If I remember correctly save_root is never set by the use explicitly, right?

Reply:
Good point! But yes, it only gets set in l. 213 using `self.save_root = os.path.join(self.root, "model_outputs")`. So we can only remove a folder called "model_outputs". But after writing this, let me just add precisely this check before the call to rmtree to make sure.

Reply:
done in 5271ea9

Reply:
now in  276f0fa because I forgot formatting